src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -49,7 +49,7 @@ public static function register($mode = false)
             'legacy' => array(),
             'other' => array(),
         );
-        $deprecationHandler = function ($type, $msg, $file, $line, $context) use (&$deprecations, $getMode) {
+        $deprecationHandler = function ($type, $msg, $file = null, $line = null, $context = array()) use (&$deprecations, $getMode) {
```

Comment:
AFAIK, only $context is "optional" (because it's use has been deprecated in 7.2)